VCFv4.3.tex

```diff
@@ -503,41 +503,100 @@ \subsection{VCF tag naming conventions}
+	\end{itemize}
+\item [\null] DEL:  Deletion relative to the reference
+	\begin{itemize}
+	\item [\null] -MEI:  Absence of a mobile element present in the reference
+		\begin{itemize}
+		\item [\null] -ALU:  Absence of a Alu mobile element present in the reference
+		\item [\null] -LINE1:  Absence of a LINE1 mobile element present in the reference
+		\item [\null] -SVA:  Absence of a SVA mobile element present in the reference
+		\item [\null] -HERV:  Absence of a HERV mobile element present in the reference
+		\end{itemize}
+	\item [\null] -RET:  Absence of a retrotransposon present in the reference
+	\end{itemize}
+\item [\null] TRA:  Translocation
+\item [\null] STR:  Short tandem repeat variation
+\item [\null] INV:  Inversion relative to the reference
+\item [\null] DUP:  Duplication
+	\begin{itemize}
+	\item [\null] TANDUP:  Tandem duplication (one of Direct, Inverted-upstream, or Inverted-downstream)
+	\item [\null] DISDUP:  Dispersed duplication
+	\end{itemize}
+\item [\null] DELINS:  Deletion-insertion (replacement of reference sequence with sequence of different content or length)
+\item [\null] OTHER:  Complex variant not covered elsewhere
+	\begin{itemize}
+	\item [\null] DEL\_DUP:  Deletion + Duplication
+	\item [\null] DEL\_INV:  Deletion + Inversion
+	\item [\null] DUP\_INV:  Duplication + Inversion
+	\item [\null] DEL\_DUP\_INV:  Deletion + Duplication + Inversion
+	\end{itemize}
+\item [\null] CNV:  Copy number variation\vspace{0.3cm}
 \end{itemize}
-
+TRA (Translocation):  A VCF translocation record describes the juxtaposition of two DNA segments from disparate genomic locations (from the same or from different chromosomes). It must include two genomic coordinates, each accompanied by an orientation (see STRAND below). A VCF translocation record only describes the immediate molecular context of the novel adjacency generated by the translocation event. It should not be confused with the traditional cytogenetic notion of translocation, which is understood to imply that the remainder of the chromosome arms beyond the specified breakpoints are also involved.\vspace{0.3cm}\\
+\setlength{\parindent}{0.3cm}
+\indent In a TRA VCF record:\vspace{0.3cm}\\
+\indent CHROM = chromosome of the first breakpoint\\
+\indent POS = coordinate of the first breakpoint\\
+\indent CHR2 = chromosome of 2nd breakpoint\\
+\indent END = coordinate of 2nd breakpoint\\
+\indent STRAND = (head-tail, tail-head, head-head, tail-tail)\\
+\indent REF = reference base at position CHROM:POS\\
+\indent ALT = reference base at position CHR2:END\vspace{0.3cm}\\
+STR (Short tandem repeat variation, simple sequence repeat, microsatellite): Short tandem repeat variants may be represented in VCF either as structural variants or as ordinary indels (see Section 5.3 for examples). The \textbf{Pindel} software package, for example, treats STRs as standard variants, while \textbf{lobSTR} treats them as structural variants. When treated as structural variants:\vspace{0.3cm}\\
```

Comment:
STR breakpoint uncertainty cannot be unambiguously represented in the current specs and it is unclear how the breakpoint uncertain due to micro-homology is handled for repeats. A two AT repeat expansion in a 10 AT unit repeat could have HOMLEN=4 and HOMSEQ=ATAT, or could be HOMLEN=20 and HOMSEQ=ATATATATATATATATATAT. Since there are 10 position the ATAT could be inserted, CIPOS should be -10,10 but it can't because CIPOS has no way of specifying that insertion can only be placed at {-10,-8,-6,-4,-2,0,2,4,6,8,10} and not at {-9,-7,-5,-3,-1,1,3,5,7,9}.